api/src/DataFixtures/MissionFixtures.php

```diff
@@ -64,132 +40,51 @@ public function load(ObjectManager $manager)
-        $mission2->setMissionType($typeInterne);
-
+        $mission->setMissionType($missionTypes->get('externe'));
         $manager->persist($mission);
-        $manager->persist($mission2);
-
-        /* Create VulnType and persist */
-        $type = new VulnType();
-        $type->setName("externe");
-        $type2 = new VulnType();
-        $type2->setName("interne");
-        $manager->persist($type2);
-        $manager->persist($type);
-
-        /* Create Impact and persist */
-
-        $impact = new Impact();
-        $impact->setName("Low");
-        $manager->persist($impact);
-
-
-        $impact2 = new Impact();
-        $impact2->setName("Medium");
-        $manager->persist($impact2);
-
-        $impact3 = new Impact();
-        $impact3->setName("High");
-        $manager->persist($impact3);
-
-        $impact4 = new Impact();
-        $impact4->setName("Critical");
-        $manager->persist($impact4);
-
-        /* Create Vulns and persist */
-        $sqli = new Vuln();
-
-        $sqliFR = new VulnTranslation();
-        $sqliEN = new VulnTranslation();
-        $sqliFR
-            ->setDescription('Injection sql')
-            ->setName('sqli');
-        $sqliFR->setLocale('fr');
-        $sqliFR->setRemediation("fait des requetes parametrer pelo");
-        $sqliEN
-            ->setDescription('Sql injection')
-            ->setName('sqli');
-        $sqliEN->setLocale('en');
-        $sqliEN->setRemediation("Use of Prepared Statements (with Parameterized Queries, you can also use of Stored Procedures and escaping All User Supplied Input");
-        $manager->persist($sqliFR);
-        $manager->persist($sqliEN);
-        $sqli->setCurrentLocale('en');
-        $sqli->addTranslation($sqliFR);
-        $sqli->addTranslation($sqliEN);
-        $sqli->setVulnType($type);
-        $sqli->setImpact($impact3);
-        $manager->persist($sqli);
-
-        $xss = new Vuln();
-
-        $xssFR = new VulnTranslation();
-        $xssEN = new VulnTranslation();
-        $xssFR
-            ->setDescription('exemple : execute /js')
-            ->setName('XSS')
-            ->setRemediation(" tu devrais mettre des headers et verifier les champs")
-            ->setLocale('fr');
-        $xssEN
-            ->setDescription('example : execute /js')
-            ->setName('XSS')
-            ->setRemediation('HTML Encode Before Inserting Untrusted Data into HTML Element Content and add some headers')
-            ->setLocale('en');
-        $xss->addTranslation($xssFR);
-        $xss->addTranslation($xssEN);
-        $xss->setVulnType($type);
-        $xss->setImpact($impact2);
-        $manager->persist($xss);
+        $this->setReference('MISSION_1', $mission);
 
+        $mission = new Mission();
+        $mission->setNessus(0);
+        $mission->addClient($clients->get('client2'));
+        $mission->setName("FAKE-MISSION-INTERNE");
+        $mission->setstartDate(\DateTime::createFromFormat('Y-m-d', "2020-11-08"));
```

Comment:
we also need to inject host into mission fixture
for exemple :  $mission2->addHost($host2); and the reference

Reply:
In the HostFixture we attach the mission to the host.
If I understand your problem a mission can have many host and a host can have many mission ? It's a many to many relationship ?

This feature can impact the frontend application.
I think you can create a new issue for analyse the impact.

Thanks